src/components/UserConfirmDialog/index.vue

```diff
@@ -59,37 +80,71 @@ export default {
     }
   },
   mounted() {
+    this.smsBtnText = this.$t('common.SendVerificationCode')
     this.$axios.get('/api/v1/authentication/confirm/', { disableFlashErrorMsg: true }).then(() => {
       this.$emit('UserConfirmDone', true)
     }).catch(err => {
       this.$log.debug('Verify otp code error: ', err)
       const backends = err.response.data.backends
       backends.sort((a, b) => b.level - a.level)
-      this.ConfirmType = backends[0].name
+      this.ConfirmType = backends[0].confirm_type
+      this.Content = backends[0].content
       if (this.ConfirmType === 'relogin') {
         this.visible = false
-        return this.$message.error(this.$t('auth.ReLoginErr'))
+        return this.$message.error(this.Content)
       }
-
       if (this.ConfirmType === 'mfa') {
-        this.Label = 'MFA'
-        this.HelpText = this.$t('common.MFARequireForSecurity')
+        this.Select = this.Content.filter(item => !item.disabled)[0].name
+        this.HelpText = this.Content.filter(item => !item.disabled)[0].placeholder
         this.visible = true
       } else if (this.ConfirmType === 'password') {
-        this.Label = this.$t('setting.password')
+        this.Select = this.$t('setting.password')
         this.HelpText = this.$t('common.PasswordRequireForSecurity')
+        this.Content = [{
+          'name': 'password'
+        }]
         this.visible = true
       }
     })
   },
   methods: {
+    helpText(val) {
+      this.HelpText = this.Content.filter(item => item.name === val)[0]?.placeholder
```

Comment:
const filter = this.Content.filter(item => item.name === val) || []
this.HelpText = filter[0]?.placeholder